vars/runPCT.groovy

```diff
@@ -10,7 +10,7 @@ def call(Map params = [:]) {
     def metadataFile = params.get('metadataFile', 'essentials.yml')
     def jenkins = params.get('jenkins', 'latest')
     def pctExtraOptions = params.get('pctExtraOptions', [])
-    def javaOptions = params.get('javaOptions', ["-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn"])
+    def javaOptions = params.get('javaOptions', [])
```

Comment:
I wouldn’t be surprised if you want to keep it here, not sure if PCT propagates this down properly but can try it and revert if needed

Reply:
Pending https://github.com/jenkinsci/plugin-compat-tester/pull/198#issuecomment-535573791 this should switch to `-ntp`.

Reply:
```suggestion
    def javaOptions = params.get('javaOptions', ['--no-transfer-progress'])
```